PhysicsTools/PatAlgos/plugins/PATPackedCandidateProducer.cc

```diff
@@ -190,7 +190,9 @@ void pat::PATPackedCandidateProducer::produce(edm::StreamID, edm::Event& iEvent,
 
     for(unsigned int ic=0, nc = cands->size(); ic < nc; ++ic) {
         const reco::PFCandidate &cand=(*cands)[ic];
+	float trkPt = cand.pt();
         float phiAtVtx = cand.phi();
+        float etaAtVtx = cand.eta();
```

Comment:
please keep in sync with 91X where trkPt and etaAtVtx are defined/initialized later [L220, 222 in this version]